src/scripts/Update.ts

```diff
@@ -1395,6 +1395,19 @@ class Update implements Saveable {
                         saveData.battleFrontier.milestones = saveData.battleFrontier.milestones.filter(milestone => milestone[1] !== name);
                     });
             }
+
+            // Rotate form IDs
+            const formIDs = [
+                // Flabebe line (swap Blue and Orange)
+                [669.02, 669.03],
+                [670.02, 670.03],
+                [671.02, 671.03],
+                // ROYGBIV Minior instead of BGIORVY
+                [774.01, 774.05],
+                [774.02, 774.04],
+                [774.03, 774.06, 774.07],
+            ];
+            Update.rotatePokemonIDs(saveData, formIDs);
```

Comment:
Had a quick look through, your rotatePokemonIDs looks fine, but this line I think should be:
```ts
        formIDs.forEach((list) => Update.rotatePokemonIDs(saveData, list));
```

On mobile so I can't test it out right now though.